tests/shakedown/tests/acceptance/test_00_setup.py

```diff
@@ -1,6 +0,0 @@
-from shakedown import *
```

Comment:
You should probably mention in the description that shakedown tests are removed (and why)